app/models/garden.rb

```diff
@@ -4,6 +4,8 @@ class Garden
   belongs_to :user
   validates_presence_of :user
 
+  after_save :reindex_guides
```

Comment:
Cool, let us know if you think of anything, so that even if you don't have time someone else can try to figure it out :)